src/v2/Apps/Partner/Components/Overview/ShowBannersRail/ShowBannersRail.tsx

```diff
@@ -170,7 +170,10 @@ export const ShowBannersRailRenderer: React.FC<
 
         return (
           <PartnerShowBannersContextProvider>
-            <ShowBannersRailFragmentContainer {...rest} {...props} />
+            <ShowBannersRailFragmentContainer
+              {...rest}
+              partner={props.partner!}
```

Comment:
Seeing you use `!` all over makes me feel a bit better about doing this with Relay components, but I still always wonder whyyy

Reply:
I don't necessarily want to here... path of least resistance. Technically this _could_ be `null` if there's an error... and you'd want to probably guard on those props specifically. If you think putting in some `// @ts-expect-error STRICT_NULL_CHECK` is clearer to future maintainers, then that's an option too.

Reply:
The "why" is mostly because of how infrequently we use non-nullable fields when defining schema in Metaphysics.